.evergreen/config.in.yml

```diff
@@ -850,7 +850,7 @@ functions:
         local_file:  src/coverage/coverage-final.json
         optional: true
         # Upload the coverage report for all tasks in a single build to the same directory.
-        # TODO NODE-3897 - change upload directory to mongo-node-driver-next
+        # TODO NODE-4707 - change upload directory to ${UPLOAD_BUCKET} 
```

Comment:
New ticket so getting mongo-node-driver-next as a new bucket doesn't hold up this PR.